### ISSUE TITLE
link integration for images

### ISSUE THREAD

**Issue by elzershark** [GeekMagicClock/smalltv-ultra#25]
Can you integrate that?

this is a saved picture:
http://192.168.68.162/set?gif=/gif/true.gif

can you integrate this?

http://192.168.68.162/set?gif=https://erforscht.com/images/true.gif
From web. not from saved pictures.

for:
http://192.168.68.162/set?gif=/gif/
and
http://192.168.68.162/set?img=/image//

thank you and bb

p.s. for test:
https://erforscht.com/images/true.gif   80x80
https://erforscht.com/images/false.gif  80x80
https://erforscht.com/images/trueb.gif  240x240
https://erforscht.com/images/falseb.gif  240x240

**Comment by GeekMagicClock** [comment]
Hi the device will not download files from internet, so that will not work.
By the way, what that for?

**Comment by elzershark** [comment]
Für smart Home. Ich nutze iobroker. In der Wetter App werden mir daten angezeigt. Ich frage, ob die Steckdose vom Warmwasserboiler (Duschwasser) an oder aus ist. Im gerät, wird mir das dann angezeigt. Oder was der Strom gerade kostet. u.s.w.


https://github.com/GeekMagicClock/smalltv-ultra/assets/429883/2036331c-43cf-4583-9425-5c2c70b7263b

**Comment by GeekMagicClock** [comment]
Yes, it could download gif from internet, but it takes time, since there's only one core, the screen will freeze when downloading, haven't decide to add this feature yet.

**Comment by Ashwagada** [comment]
> Yes, it could download gif from internet, but it takes time, since there's only one core, the screen will freeze when downloading, haven't decide to add this feature yet.

Maybe you can add this and optimize it in the future.

**Comment by elzershark** [comment]
> Ja, es könnte GIF aus dem Internet herunterladen, aber es braucht Zeit, da es nur einen Kern gibt und der Bildschirm beim Herunterladen einfriert. Ich habe mich noch nicht entschieden, diese Funktion hinzuzufügen.

> Yes, it could download gif from internet, but it takes time, since there's only one core, the screen will freeze when downloading, haven't decide to add this feature yet.


Perfect would be: download and display. and delete it again immediately. The display still remains as long as there is no reboot or something like that. gif file only because it is small. without animation.

**Comment by elzershark** [comment]
> Yes, it could download gif from internet, but it takes time, since there's only one core, the screen will freeze when downloading, haven't decide to add this feature yet.

and give me the firmware to test, please 😁😁
